src/components/elements/TimePicker.js

```diff
@@ -17,7 +18,26 @@ class TimeAndDatePicker extends Component {
 
   constructor () {
     super()
-    this.state = { startDate: null, startTime: null, endDate: null, endTime: null }
+    this.state = {
+      endDate: DateTime.local().toISODate().toString(),
```

Comment:
I _think_ we should use `utc` unless the local zone is important for some reason.